.github/workflows/linter.yml

```diff
@@ -4,51 +4,51 @@ jobs:
   Bandit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
         with:
           python-version: 3.8
 
       - name: Run checks
         run: |
-          pip install --user -r <(grep "^bandit" ./requirements.txt)
+          pip install --user -r tests/requirements.txt
```

Comment:
I think that this can increase the network cost very much. It is because `tfds` depends on `tensorflow` and it tries to install them for all jobs repeatedly.

Reply:
If so, do you think it's better to install only the required thing for each step like former one?

Reply:
Guess it would be better to split tests/requirements.txt to tests/requirements_lint.txt (light linting packages) and tests/requirements.txt (heavy one for pytest). Then, tests/requirements.txt includes tests/requirements_lint.txt.

Reply:
That's good idea. I will push them. Thanks.

Reply:
one thing need to check. is it okay to have "requirements" subfolder on the root of repo to hold all requirements txt files together?